src/main/kotlin/bsu/cc/parser/Actions.kt

```diff
@@ -91,7 +115,7 @@ fun writeWorkbook(workbook: XSSFWorkbook, fileName: String) {
 fun identifyAndWriteConflicts(fileName: String, config: ConfigProperties, sheetIndex: Int = 0) : String {
```

Comment:
Looking back  on this. It feels weird to have something as broad (and needlessly specific to tornado) config properties instead of just the constraints file path. I think having this take the file path would make more sense (and this is also what I have done as part of adding the constraints file drop down